attesters/tsm/tsm_test.go

```diff
@@ -54,3 +59,70 @@ func Test_GetSupportedFormats(t *testing.T) {
 
 	assert.Equal(t, expected, p.GetSupportedFormats())
 }
+
+func Test_GetEvidence_wrong_nonce_size(t *testing.T) {
+	inblob := []byte("abcdefghijklmnopqrstuvwxyz123456")
+	in := &compositor.EvidenceIn{
+		ContentType: string("mediaType"),
```

Comment:
nit: ContentType could be a valid value, such as ApplicationvndVeraisonConfigfsTsmJson, as the test is for incorrect nonce size.

Reply:
Fixed